### ISSUE TITLE
Error when trying to close FloatingView by dragging it to bottom

### ISSUE THREAD

**Issue by icoolguy1995** [recruit-lifestyle/FloatingView#42]
Hi! First of all thanks for this great library, it's really awesome to use. However, when I try to drag chatHead(Floating view) to bottom(to close it) and after Floating View closes I get the following error:

```
java.lang.IllegalArgumentException: View=jp.co.recruit_lifestyle.android.floatingview.FloatingView{79038e7 G.E...... ......ID 0,0-78,78} not attached to window manager
                                                                         at android.view.WindowManagerGlobal.findViewLocked(WindowManagerGlobal.java:473)
                                                                         at android.view.WindowManagerGlobal.updateViewLayout(WindowManagerGlobal.java:368)
                                                                         at android.view.WindowManagerImpl.updateViewLayout(WindowManagerImpl.java:101)
                                                                         at jp.co.recruit_lifestyle.android.floatingview.FloatingView.updateViewLayout(FloatingView.java:507)
                                                                         at jp.co.recruit_lifestyle.android.floatingview.FloatingView.onUpdateSystemLayout(FloatingView.java:418)
                                                                         at jp.co.recruit_lifestyle.android.floatingview.FloatingViewManager.onScreenChanged(FloatingViewManager.java:234)
                                                                         at jp.co.recruit_lifestyle.android.floatingview.FullscreenObserverView.onSystemUiVisibilityChange(FullscreenObserverView.java:129)
                                                                         at android.view.View.dispatchSystemUiVisibilityChanged(View.java:20483)
                                                                         at android.view.ViewRootImpl.handleDispatchSystemUiVisibilityChanged(ViewRootImpl.java:5622)
                                                                         at android.view.ViewRootImpl$ViewRootHandler.handleMessage(ViewRootImpl.java:3674)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                         at android.os.Looper.loop(Looper.java:154)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:6128)
                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)

```

**Comment by YoshihideSogawa** [comment]
I can not confirm this bug.
However, I fixed a code that is likely to cause problems.
Please check whether the same bug occurs on [ver 2.2.2-alpha](https://github.com/recruit-lifestyle/FloatingView/releases/tag/2.2.2-alpha).

`compile 'com.github.recruit-lifestyle:FloatingView:2.2.2-alpha'`

**Comment by YoshihideSogawa** [comment]
Waiting for 2.2.2

**Comment by YoshihideSogawa** [comment]
Fixed [2.2.2](https://github.com/recruit-lifestyle/FloatingView/releases/tag/2.2.2)
